web/src/components/PoiDetails.tsx

```diff
@@ -53,8 +52,8 @@ const Thumbnail = styled('img')`
   }
 `
 
-const Distance = styled('div')`
-  ${helpers.adaptiveFontSize};
+export const StyledTypography = styled(Typography)<TypographyProps>`
```

Comment:
❓ Does this need to be exported?

Reply:
I reused it in another component, but it could be a bad practice ...